ACE/ace/Shared_Memory_Pool.h

```diff
@@ -182,14 +184,16 @@ class ACE_Export ACE_Shared_Memory_Pool : public ACE_Event_Handler
   /// Base shared memory key for the segment.
   key_t base_shm_key_;
 
+  /// Base shared memory id
+  int base_shm_id_ {};
```

Comment:
Is this needed as a separate variable or will it always be available in the SHM_TABLE?

Reply:
Gone in the last revision